Documentation/GeneralConventions/GuidesXml.rst

```diff
@@ -0,0 +1,199 @@
+..  _guides-xml:
+
+================================
+:file:`Documentation/guides.xml`
+================================
+
+This XML file contains meta information and configuration used during rendering
+of a manual.
+
+Example:
+
+..  literalinclude:: _guides-simple.xml
+    :capition: Documentation/guides.xml
+
+..  hint::
+    If you are migrating from the legacy Sphinx-based rendering and still have
+    a :file:`Documentation/Settings.cfg` you can use an automatic migration
+    tool to :ref:`migrate the settings.cfg into a guides.xml <migrate_guides_xml>
+
+.. _settings-guides:
+
+The `<guides>` tag
+==================
+
+It is used for general configuration during parsing and rendering.
```

Comment:
What does `links-are-relative="true"` mean?

Reply:
Wether relative or absolute links are used in the HTML result, however that is something that has to be documented in the Guides